### PR TITLE
Foo

### PR DESCRIPTION
**Pull request by jaredm4**
This is ```neat```

### CODE REVIEW

**Comment by AlexJuarez** [comment]
garbage code

**Comment by AlexJuarez** [comment]
never going to merge

File: bar [hunk 1]
```diff
@@ -0,0 +1 @@
+baz
```
Comment (AlexJuarez) [inline bar]: why?

Comment (ouellettetech) [inline bar]: This looks like it would blow up the world....

Comment (jaredm4) [inline bar]: because :beers:

Comment (ouellettetech) [inline bar]: :-1:

**Comment by jaredm4** [comment]
gcn5wwt

File: bar (hunk 1 above)
Comment (jaredm4) [inline bar]: fine I'll remove it

**Comment by jaredm4** [comment]
#1

**Pull request merged**